web-animations-1/Overview.bs

```diff
@@ -2465,123 +2465,203 @@ Animation effects {#animation-effects}
   An <dfn>animation effect</dfn> is a [=timing node=]
   that provides a static description of some timed behavior.
 
-### Relationship between animation effects and animations ### {#animation-effects-and-animations}
+  All [=animation effects=] have the following properties:
+  * an [=active interval=],
+      determined by its [=start delay=] and [=active duration=],
+      defining the time interval during which its behavior takes effect
+  * an [=iteration duration=], [=iteration count=], and [=iteration start=]
+      defining how its behavior repeats within that interval
+  * a [=playback direction=] applying to each iteration,
+      to allow reversing the timing of effects within an iteration
+  * a [=fill mode=] defining the application of effects
+      [=before phase|before=]/[=after phase|after=] its [=active interval=]
+  * optionally, an [=animation effect/associated animation=]
+      and through it, optionally, an [=associated with a timeline|associated timeline=],
+      which cause the [=animation effect=] to apply its behavior
+      by driving its [=local time|local time value=]
 
-  The [=associated effect=] of an [=animation=], if set,
-  is a type of [=animation effect=].
-  The [=associated effect=] of an [=animation=]
-  is said to be <dfn lt="associated with an animation">associated</dfn>
-  with that animation.
+### Types of animation effects ### {#types-of-animation-effects}
+
+  This specification defines a single type of [=animation effect=]:
+  [=keyframe effects=].
+  Subsequent levels of this specification
+  will define further types of [=animation effects=].
+
+### Associated Animation and Timeline ### {#animation-effects-and-animations}
```

Comment:
Nit: capitalization

```suggestion
### Associated animation and timeline ### {#animation-effects-and-animations}
```

Reply:
Hah. I'm so used to using title case. :)